cfme/tests/intelligence/reports/test_widgets.py

```diff
@@ -71,7 +71,7 @@ def _finalize():
         with update(dashboard):
             dashboard.widgets = default_widgets
     request.addfinalizer(_finalize)
-    dashboard.reset_widgets()
+    DefaultDashboard.reset_widgets()
```

Comment:
I guess only a one default dashboard can exist. In this test we work with the default dashboard, so the class method can be used here.